nni/retiarii/converter/graph_gen.py

```diff
@@ -557,7 +547,8 @@ def _handle_inputchoice(self, module):
     def _handle_valuechoice(self, module):
         return {
             'candidates': module.candidates,
-            'label': module.label
+            'label': module.label,
+            'accessor': module._accessor
```

Comment:
Could you explain the idea of accessor?

Reply:
Please refer to newly-added comments and response to Quanlu.